src/handbook/company/strategy.md

```diff
@@ -35,16 +35,26 @@ Software plays a key part in education, reducing vendor lock-in, and community b
 
 ## The market
 
-Corporations that have many data sources with unpredictable structure.
+Our Target persona is the Industrial engineer: specifically those working in manufacturing, logistics, and other data-intensive industrial sectors where operational efficiency is equivalent to generating profit and creates real business value.
 
-The ideal lead has adopted Node-RED already and is in the following industries:
-1. Manufacturing (IIoT & Industry 4.0)
-1. Logistics
-1. Agriculture
-1. Building management
-1. Mining, Oil & Gas, and renewable energy
+We target them because Industrial engineers are the driving force behind innovations that lead to process optimizations in their organizations. They possess deep domain expertise and understand the intricacies of their operations. In order to improve their systems, they need to actively collect and analyze data.
 
-Due to the breadth of the Node-RED community and the industries they
-apply it in, FlowFuse does not qualify customers based on industry.
+## The problems 
```

Comment:
One of the biggest problems that @Steve-Mcl pointed out to me before is legacy hardware that makes it very hard to extract data Node-RED and it's many integrations are what are making that possible, and modernising legacy hardware.

Reply:
I interpret "silo'd data" as an integration challenge, not an extraction challenge